src/Book.java

```diff
@@ -0,0 +1,48 @@
+public class Book {
```

Comment:
Nice class! Very well organized. One note: your comments are not in the official documentation comment format. Ideally, comments that describe methods should be in the following format:

```java
/**
 * Method description
 */
public void method(int p1) {
    //stuff
}
```

this special format is what tells VS Code and other IDEs that this is the description for the entire method and not some random comment